### ISSUE TITLE
Own Sync server Option?

### ISSUE THREAD

**Issue by leebrand** [mozilla/policy-templates#367]
Hello,

i would like set my own sync server address/port for Firefox, i can do that over config but
for all users it isnt praktikable.

Thanky.

**Comment by mkaply** [comment]
Are you asking if you can host your own server or simply change the URLs? Have you built your own server?

**Comment by leebrand** [comment]
yes, i have docker and can run my own server as dockerimage on synology nas.

**Comment by mkaply** [comment]
> yes, i have docker and can run my own server as dockerimage on synology nas.

So you have been able to get it to work? Other people have not been able to. Can you detail all the different URL preferences you had to change that you want in policy?

**Comment by leebrand** [comment]
i have tested it for months and stopped because i wont configure each pc.
my network confugurantions changes frequently, so i wont run every time i chance settings to configure clients.
Today i dont use sync server, because i dont know about security of sync servers and i dont give any data out of my house.
https://hub.docker.com/r/sunx/mozilla-syncserver/

**Comment by mkaply** [comment]
> Can you detail all the different URL preferences you had to change that you want in policy?

In particular, did you have a login and a sync server?

**Comment by leebrand** [comment]
this settings are relevant for policy: (black mark)
![grafik](https://user-images.githubusercontent.com/49483694/57061113-0f6d8d00-6cbc-11e9-8b67-8710179dfc44.png)
other settings above maybe interessant for users with his own account server.
futher nice to have options to select what i want to sync.
![grafik](https://user-images.githubusercontent.com/49483694/57061350-cec24380-6cbc-11e9-9f42-f51504364845.png)
thanky.
